### ISSUE TITLE
No match for method kex algos

### ISSUE THREAD

**Issue by mppben** [blinksh/blink#711]
I'm using version 12.8 from the Apple App Store, and get the following when attempting to ssh into a Bluehost server.  Can you advise on how to get around this, or when it could be fixed?

```
blink> ssh sc

kex error : no match for method kex algos: server [diffie-hellman-group-exchange-sha256], client [curve25519-sha256,curve25519-sha256@libssh.org,ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521,diffie-hellman-group16-sha512,diffie-hellman-group18-sha512,diffie-hellman-group14-sha1,diffie-hellman-group1-sha1]
```

**Comment by yury** [comment]
Hi @mppben,

Current libssh release version (0.8) doesn't support it. But there is diffie-hellman-group-exchange-sha256 algorithm in master branch.
So we are waiting for libssh 0.9.0 to release. 
Can't estimate here, it should be release a month ago. 

Refs #709, #616

**Comment by yury** [comment]
Hi @mppben.

Should be fixed in 12.9. I'm closing this issue. Feel free to reopen.

**Comment by mppben** [comment]
Thanks, @yury.  That's fixed it for me!
